src/org.xtuml.bp.io.core/arc/export_functions.inc

```diff
@@ -1108,6 +1108,8 @@ throws IOException
     	// get the body
     	String body = Ooaofooa.Get_formatted_body( inst.getModelRoot(), inst, false );
     	if ( !body.isEmpty() ) {
+        // wrap with body delimiters
+        body = "//! ACTIVITY START DO NOT EDIT THIS LINE\n" + body.trim() + "\n//! ACTIVITY END DO NOT EDIT THIS LINE\n\n";

```

Comment:
The text is the old style.  Change START to BEGIN and add perionds

Reply:
done